### PR TITLE
doc auditLog filter

### LINKED ISSUE #798: auditLog() filter captures the body in logs
https://github.com/zalando/skipper/blob/master/filters/log/log.go#L157-L162 shows the relevant code.
Default is set to 1024 https://github.com/zalando/skipper/blob/master/cmd/skipper/main.go#L43
This can be overidden via  `--max-audit-body=<int>`

I learned that this was done on purpose, but maybe we should change the default.
Therefore I will remove the **bug** label.

### PR DESCRIPTION
**Pull request by szuecs**
doc for auditLog() filter, fixes #798

Signed-off-by: Sandor Szuecs <sandor.szuecs@zalando.de>

### CODE REVIEW

File: docs/reference/filters.md [hunk 1]
```diff
@@ -803,3 +803,15 @@ Example:
 ```
 enableAccessLog()
 ```
+
+## auditLog
+
+Filter `auditLog()` logs the request and N bytes of the body into the
+log file. N defaults to 1024 and can be overidden with
+`--max-audit-body=<int>`. `N=0` omits logging the body.
```
Comment (aryszka) [inline docs/reference/filters.md]: single leading dash: `-max-audit-body=<int>`

Comment (szuecs) [reply docs/reference/filters.md]: fixed

**Comment by szuecs** [comment]
:+1:

**Comment by szuecs** [comment]
@aryszka I fixed your commnts

**Comment by aryszka** [comment]
:+1:

**Pull request merged**